source/features/latest-tag-button.tsx

```diff
@@ -102,7 +97,7 @@ async function init(): Promise<false | void> {
 		</a>
 	);
 
-	breadcrumb.after(link);
+	select('#branch-select-menu')!.parentElement!.after(link);
```

Comment:
This element always exists, no need to check

Reply:
Looks right. We used to use an element that sometimes didn't exist; the selector was changed but not the logic: https://github.com/sindresorhus/refined-github/pull/3353/files